sklearn/ensemble/gradient_boosting.py

```diff
@@ -157,7 +197,29 @@ def predict(self, X):
 
 
 class LogOddsEstimator(object):
-    """An estimator predicting the log odds ratio."""
+    """An estimator predicting the log odds ratio.
+
+    This estimator takes only the binary target labels of the training set into
+    account. The prediction is independent of the X values in the training set
+    and those passed to the :func:`predict` function.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.ensemble import gradient_boosting
+    >>>
+    >>> y = np.array([1, 1, 0, 1])
+    >>>
+    >>> est = gradient_boosting.LogOddsEstimator()
+    >>> est.fit(np.random.rand(5, 2), y)
+    >>> print(est.prior)
```

Comment:
no need to use `print`, `est.prior` prints the value in the interpreter, this applies to the other examples as well.

Reply:
done